src/components/TileVideo/TileVideo.js

```diff
@@ -0,0 +1,20 @@
+import { useMediaTrack } from '@daily-co/daily-react-hooks';
+import { memo, useEffect, useRef } from 'react';
+
+const TileVideo = memo(function ({ id, isScreenShare }) {
+  const videoTrack = useMediaTrack(id, isScreenShare ? 'screenVideo' : 'video');
+
+  const videoElement = useRef(null);
+
+  useEffect(() => {
+    /*  The track is ready to be played. We can show video of the remote participant in the UI.*/
```

Comment:
Nit: we are also using this component for rendering local video, so maybe remove 'remote'!

Reply:
eee I always forget to double check comments 😇 thanks for that catch